Sources/View/ViewModelView.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ViewModelView.swift
+//  Alicerce
+//
+//  Created by André Pacheco Neves on 05/05/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public protocol ViewModelView: View {
+    associatedtype ViewModel
+
```

Comment:
On the other side, having it like this also allows injecting via property, which may sometimes be useful. 👍